gapis/api/vulkan/replay.go

```diff
@@ -781,13 +783,15 @@ type profileRequest struct {
 	overrides    *path.OverrideConfig
 	traceOptions *service.TraceOptions
 	handler      *replay.SignalHandler
+	buffer       *bytes.Buffer
 }
 
 func (a API) GetInitialPayload(ctx context.Context,
 	capture *path.Capture,
 	device *device.Instance,
 	out transform.Writer) error {
 	transforms := transform.Transforms{}
+	transforms.Add(&profilingLayers{})
```

Comment:
This transform should only be added if we're profiling **and** if the layer is available.

Reply:
Oops, actually didn't mean to leave it in.  For now, replay profiling will not use split replays